### DISCUSSION TITLE
How to detect if an image is removed

### DISCUSSION THREAD

**Discussion by GNITOAHC** [mdx-editor/editor#287] [Q&A]
Hi, thanks for this amazing editor! I'm currently working on a project with online editor. Whenever an image is dropped, it gets uploaded, but this rapidly increases the size of the database. I'm wondering how I can detect if an image has been removed from the editor, so that I can simultaneously remove it from the remote database.

Thanks!!

**Comment by jaxomlotus** [comment]
Here's an approach I would take:
Create an empty array and each time an image is added to the editor, simultaneously add it to the array.
When you finalize your code to add to the database - for example, if a user hits Save - loop through your array and see if the text in the array can still be found in the document. If not, then queue off a call to the database to delete it.
